src/redux/users/usersSlice.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable */
+// import { createSlice } from "@reduxjs/toolkit";
+// import { createAsyncThunk } from "@reduxjs/toolkit";
+
+// const initialState = {
+//     loading: false,
+//     users: [],
+//     error: ''
+// }
+
+// const UserSlice = createSlice({
+//     name: 'user',
+//     initialState,
+// })
```

Comment:
- [ ] Kindly avoid leaving commented pieces of code in your codebase. It makes your codebase unclean and is not a good practice :+1: